src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataParser;
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.FileWriter;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.impl.DataParserImpl;
+import core.basesyntax.service.impl.FileReaderImpl;
+import core.basesyntax.service.impl.FileWriterImpl;
+import core.basesyntax.service.impl.ReportGeneratorImpl;
+import core.basesyntax.service.impl.ShopServiceImpl;
+import core.basesyntax.strategy.FruitStrategy;
+import core.basesyntax.strategy.FruitStrategyImpl;
+import core.basesyntax.strategy.operation.BalanceOperation;
+import core.basesyntax.strategy.operation.OperationHandler;
+import core.basesyntax.strategy.operation.PurchaseOperation;
+import core.basesyntax.strategy.operation.ReturnOperation;
+import core.basesyntax.strategy.operation.SupplyOperation;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String FILENAME_TO_READ = "reportToRead.csv";
+    private static final String FILENAME_TO_WRITE = "finalReport.csv";

```

Comment:
The file paths should not be hardcoded in the `Main` class. It's better to accept them as command-line arguments or read them from a configuration file to avoid recompilation when the file paths change.